doc/nrf/releases_and_maturity/releases/release-notes-0.4.0.rst

```diff
@@ -186,7 +186,7 @@ Documentation
   * :ref:`dk_buttons_and_leds_readme`
   * :ref:`lib_nfc_ndef`
   * :ref:`app_event_manager` and :ref:`nrf_profiler`
-  * :ref:`at_cmd_parser_readme`, :ref:`at_params_readme`, and :ref:`modem_info_readme`
+  * AT command parser, AT params, and :ref:`modem_info_readme`
```

Comment:
AT parameters library still remains functional, right? If yes, you can leave the reference to it as is.

Reply:
No, it's used in conjunction with the AT command parser

Reply:
At least AT parameters then.

Reply:
Ok